contentcuration/contentcuration/frontend/shared/mixins/draggable/constants.js

```diff
@@ -5,3 +5,33 @@ export const DraggableTypes = {
   COLLECTION: 'collection',
   REGION: 'region',
 };
+
+const { ITEM, COLLECTION, REGION, CONTAINER } = DraggableTypes;
+
+export const DraggableContainerTypes = { ITEM, COLLECTION, REGION, CONTAINER };
+
+export const DraggableSearchOrder = [ITEM, COLLECTION, REGION, CONTAINER];
+
+/** @see https://developer.mozilla.org/en-US/docs/Web/API/DataTransfer/dropEffect */
+export const DropEffect = {
+  COPY: 'copy',
+  MOVE: 'move',
+  NONE: 'none',
```

Comment:
Should this be the default DropEffect similar to how there's a default DragEffect?

Reply:
Yeah that's a good idea. I'm really not sure why I defaulted it to copy!